docs/application/dotnet/guides/nui/linear-layout.md

```diff
@@ -0,0 +1,128 @@
+# Linear Layout
+
+## Overview
+
+`LinearLayout` is a linear box layout, automatically handling children laid out vertically or horizontally. You can set the layout direction vertically or horizontally by using `LinearOrientation` property.
+
+<table style="width:100%">
+<tr>
+<td style="width:100%" align="center">
+<img src="./media/linearTotal.png">
+</td>
+</tr>
+</table>
+
+Positioning children in a linear form, one after the other, `CellPadding` can be provided to insert a space between each child. Unlike generic padding, this will not result in a space at the start, end or top, bottom of the layout.
+
+`LinearAlignment` defines where the children should start being positioned from, and it's very useful if the children do not use all the space of the parent.
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LinearOrientation`    | LinearLayout.Orientation | Gets/Sets Orientation of the linear layout : vertical or horizontal |
+| `LinearAlignment`      | LinearLayout.Alignment  | Gets/Sets the global child alignment to be used |
+| `CellPadding`          | Size2D      | Gets/Sets Spacing between the cells : horizontal space and vertical space |
+
+
+## Orientation
+
+A `LinearOrientation` indicates the direction of children layouts : horizontal or vertical. The default value is Horizontal.
+
+| Horizontal | Vetical |
+| -----------------------| --------------- |
+| ![Horizontal](./media/horizontalLayout.png) | ![Vertical](./media/verticalLayout.png)  |
+
+```csharp
+View layoutView = new View();
+var linearLayout = new LinearLayout();
+linearLayout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+layoutView.Layout = layout;
+```
+
+
+## Alignment
+
+A `LinearAlignment` handles how Layout items are positioned within their parent Layout in the space. By default, items are aligned at the Top and Begin.
```

Comment:
Line 44 and Line 55 should be "Begin" because it's one of 'Alignment' enumerations.
On the other hand, line 15 is just to describe a position of padding. Especially, in NUI, the type of Padding is 'Extents' and it uses "start", "end", "top", and "bottom".

The concepts of "Begin" and "start" are same, but "Begin" is a name of enum.
So, if they confuse users with definition, I'll add "Extents" in CellPadding.